bower.json

```diff
@@ -9,31 +9,31 @@
     "tests"
   ],
   "dependencies": {
-    "jquery": "2.1.1",
-    "underscore": "1.8.3",
-    "backbone": "1.2.3",
-    "backbone-associations": "0.6.2",
-    "bootstrap": "3.3.5",
-    "font-awesome": "4.4.0",
-    "requirejs": "2.1.20",
-    "requirejs-text": "2.0.14",
+    "jquery": "^3.2.1",
```

Comment:
@sisiz if moving to yarn is a separate task then I think the `^` should be removed so deployments are still consistent